pkg/repositories/gormimpl/project_repo.go

```diff
@@ -62,6 +62,41 @@ func (r *ProjectRepo) ListAll(ctx context.Context, sortParameter common.SortPara
 	return projects, nil
```

Comment:
could we update the call to ListAll in cluster resource controller to use the new List and delete the ListAll method if it's not used anywhere else? and also add a filter for non-archived projects in that call too?

Reply:
Yeah the last commit does that but I haven't deleted the function and mock. Now I see this makes sense so I can move on cleaning up. Thanks.